modules/core/nuxeo-core-api/src/test/java/org/nuxeo/ecm/core/api/security/TestACL.java

```diff
@@ -46,6 +48,16 @@ public void testGetName() {
         assertEquals("test acl", acl.getName());
     }
 
+    @Test
+    public void testExceptionOnNullName() {
+        try {
+            acl = new ACLImpl(null, false);
+            fail("ACL is not supposed to be created.");
+        } catch (NuxeoException e) {
+            // Test ok
```

Comment:
Please assert on the message of the exception too:

    assertEquals("ACL name can't be null", e.getMessage());